roles/storage-cns/README.md

```diff
@@ -0,0 +1,11 @@
+# Ansible role: storage-cns
+
+This role deploys cluster resources necessary for kubevirt to interface with
+CNS (glusterfs) storage.  This role assumes that CNS itself has already been
+installed.
+
+## Configuration parameters
```

Comment:
nit: Can you make this a table like the other READMEs

Reply:
Sure.